tests/stubtest_allowlists/py3_common.txt

```diff
@@ -606,6 +606,7 @@ wsgiref.handlers.BaseHandler.status
 # Iterable classes that don't define __iter__ at runtime (usually iterable via __getitem__)
 # These would ideally be special-cased by type checkers; see https://github.com/python/mypy/issues/2220
 ctypes.Array.__iter__
+docutils.nodes.Element.__iter__
```

Comment:
This needs to be added to `stubs/docutils/@tests/stubtest_allowlist.txt` rather than the stdlib allowlist